### PR TITLE
Document the GET shifts/count endpoint

### PR DESCRIPTION
**Pull request by curtis1000**
Refs WEB-4816

Here's the corresponding app PR: https://github.com/wheniwork/wheniwork-app/pull/3920

### CODE REVIEW

File: source/methods/shifts.md.erb [hunk 1]
```diff
@@ -515,3 +515,62 @@ Key | Description
...
+{
+    "shifts":17,
+    "locations":1,
+    "users":8,
+    "positions":3,
+    "sites":0
+}
+<% end %>
+
+This method allows you to count shifts as well as their associated locations, users, positions, and sites. The accepted parameters are identical to the list shifts endpoint. See the parameters below for all the options.
```
Comment (bvisness) [inline source/methods/shifts.md.erb]: @curtis1000 Let's make "list shifts" a link to the List Shifts endpoint.

**Comment by bvisness** [comment]
Other than that one nitpick, 👍 ! I'll merge and deploy it once WEB-4816 is deployed.

**Comment by curtis1000** [comment]
Added that link @bvisness

**Comment by bvisness** [comment]
@curtis1000 This change has been deployed.

**Pull request merged**